HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -399,6 +408,7 @@ void HLTScoutingEgammaProducer::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<edm::InputTag>("DphiMap", edm::InputTag("hltEgammaGsfTrackVars:Dphi"));
   desc.add<edm::InputTag>("MissingHitsMap", edm::InputTag("hltEgammaGsfTrackVars:MissingHits"));
   desc.add<edm::InputTag>("OneOEMinusOneOPMap", edm::InputTag("hltEgammaGsfTrackVars:OneOESuperMinusOneOP"));
+  desc.add<edm::InputTag>("fBremMap", edm::InputTag("hltEgammaGsfTrackVars:fbrem"));
```

Comment:
According to your comment:
```
process.hltScoutingEgammaPacker.fBremMap = cms.InputTag( 'hltEgammaGsfTrackVarsUnseeded','fbrem' )
```
the proper value to be put here is
```
cms.InputTag( 'hltEgammaGsfTrackVarsUnseeded','fbrem' )
```
Given this, a coding of a customisation routine is not advised. 

As Marco said, you might reconsider the values you put for the other InputTags.

Reply:
@mmusich @Martin-Grunewald The default values are a relic of the old menu when it was built. I will change the default values in the description and test it. For now I will skip the customisation. Thank you for the insight.